src/components/Chat.js

```diff
@@ -255,15 +265,21 @@ export default {
 			},
 			showExistingProof: function() {
 				let hash = this.proof.hash;
-				this.addMessageDelayed({
-					sender: MessageSenderEnum.APP,
-					body: {
-						type: MessageBodyTypeEnum.LINK,
-						description: "This file has already been notarized",
-						title: hash,
-						url: this.$router.resolve('/proofs/' + hash).href
-					},
-				}, this.defaultDelay, true);
+
+        this.getIpfsContent(this.proof.ipfsHash, (image)=>{
+          console.log('ipfs: ' + image);
+          this.addMessageDelayed({
+              sender: MessageSenderEnum.APP,
+              body: {
+                type: MessageBodyTypeEnum.LINK,
+                description: "This file has already been notarized",
+                title: hash,
+                url: image
```

Comment:
this should link to the proof detail, an image is not needed here

Reply:
fixed